docs/pages/deployment/cli-reference.rst

```diff
@@ -129,8 +134,11 @@ Imports private keys from filesystem based storage into Vault. The given directo
       --storage.redis.password string                 Redis database password. If set, it overrides the username in the connection URL.
       --storage.redis.username string                 Redis database username. If set, it overrides the username in the connection URL.
       --strictmode                                    When set, insecure settings are forbidden.
+      --tls.certfile network.enabletls                PEM file containing the certificate for the server (also used as client certificate). Required when network.enabletls is `true`.
       --tls.certheader string                         Name of the HTTP header that will contain the client certificate when TLS is offloaded.
+      --tls.certkeyfile tls.enable                    PEM file containing the private key of the server certificate. Required when tls.enable is `true`.
       --tls.offload tls.certheader                    Whether to enable TLS offloading for incoming connections. If enabled tls.certheader must be configured as well.
+      --tls.truststorefile string                     PEM file containing the trusted CA certificates for authenticating remote servers. (default "truststore.pem")
```

Comment:
incorrect flag types

Reply:
?

Reply:
this is related to the other comment in core/server_config.go